src/loaders/genome_collection/parse_computed_genome_attribs.py

```diff
@@ -1,16 +1,16 @@
 """
 PROTOTYPE
 
-This script involves processing computed genome attributes and organizing them into a structured format suitable
-for importing into ArangoDB. The resulting JSON file will be used to update (overwrite/insert) the database with the parsed
-genome attribute data.
+This script involves processing tool result files and organizing them into a structured format suitable
+for importing into ArangoDB. The resulting JSON file will be used to update (overwrite/insert) the database with the
+parsed data.
 
 Note: If the ArangoDB collection has been previously created using a JSON file generated by the genome attributes
       loader script and if you want to replace the data created by that loader in ArangoDB, it is crucial to ensure
       that the arguments "--load_ver" and "--kbase_collection" are consistent with the ones used in the genome
       attributes loader script in order to ensure that the same key is generated for the corresponding Arango document.
 
-usage: parse_computed_genome_attribs.py [-h] --load_ver LOAD_VER --kbase_collection KBASE_COLLECTION
+usage: parse_tool_results.py [-h] --load_ver LOAD_VER --kbase_collection KBASE_COLLECTION
```

Comment:
I will do the actual rename right after this PR. Otherwise, it's hard to tell the diff if I do it all together.